cms/static/js/views/pages/paged_container.js

```diff
@@ -28,11 +28,13 @@ define(['jquery', 'underscore', 'gettext', 'js/views/pages/container', 'js/views
 
             refreshXBlock: function(element, block_added, is_duplicate) {
                 var xblockElement = this.findXBlockElement(element),
-                    rootLocator = this.xblockView.model.id;
+                    rootLocator = this.xblockView.model.id,
+                    renderParameters;
                 if (xblockElement.length === 0 || xblockElement.data('locator') === rootLocator) {
                     this.render({refresh: true, block_added: block_added});
                 } else {
-                    this.refreshChildXBlock(xblockElement, block_added, is_duplicate);
+                    renderParameters = {can_edit_visibility: false};
```

Comment:
@cahrens That might be possible, but I think it would require a different (possibly more involved) approach: The view defined in library_container (`LibraryContainerView`) extends the view defined in views/paged_container (`PagedContainerView`), not the one defined in this file. So it's not a matter of just overriding the `refreshXBlock` function in `LibraryContainerView`.

Additionally, the code in `PagedContainerView` is not (directly) involved in rendering a newly created XBlock; this part of the process is handled by pages/container (`XBlockContainerPage`) and this file (`PagedXBlockContainerPage`), as well as the generic `XBlockView`. Control only moves to `PagedContainerView` after the new XBlock has been rendered, to update the current page if necessary.

So if we wanted to handle disabling of visibility settings in `LibraryContainerView` we wouldn't be able to stick with the current approach; that step would have to be handled entirely on the client somehow. I'm not sure if it would it make sense for `LibraryContainerView` to be responsible for that.